config/chime_science_run_gpu.yaml

```diff
@@ -1537,9 +1537,9 @@ extra_tracking_beams:
     beam_send_11:
       kotekan_stage: bufferSend
       buf: beam_buffer_11_merged
-      # Moose
-      server_ip: 10.1.50.10
-      server_port: 11030
+      # outrigger-gating
+      server_ip: 10.1.50.12
+      server_port: 68
```

Comment:
I think this is the wrong beam (and also the wrong comment in the config here.   `moose` was replaced with (or became) `bass`.   You want to use beam `10` which is currently pointed at the Absorber search machine.

Reply:
Also the IP address for `outrigger-gating` on the 10.1/16 network is `10.1.50.52`

Reply:
Yeah, you're right. `10.1.50.10` is `bass`. I updated the comment and made the config changes to beam `10`.